static/src/stylesheets/module/content/_content.scss

```diff
@@ -666,7 +666,7 @@
         overflow: visible;
     }
 
-    button {
+    .button {
```

Comment:
Are you sure about this? I think the `button` needs to be there as we need an over qualified selector in certain situations. How about: `button, .button { /stuff/ }`?